messaging/src/main/java/org/axonframework/messaging/eventhandling/processing/streaming/pooled/Coordinator.java

```diff
@@ -332,7 +332,7 @@ private boolean initializeTokenStore() {
                     }
             ));
         } catch (Exception e) {
-            logger.info(
+            logger.error(
```

Comment:
As the description states, this is not really an error, I suggest setting it to warn instead.